storage/innobase/dict/dict0stats.cc

```diff
@@ -1372,6 +1379,77 @@ static dberr_t btr_pcur_open_level(btr_pcur_t *pcur, ulint level, mtr_t *mtr,
 }
 
 
+/** Stores various statistical metrics gathered during an index scan
+or fetch operation. */
+struct IndexScanStatistics
+{
+  /* Array for number of distinct keys for all prefixes */
+  ib_uint64_t *n_diff_on_level = nullptr;
+
+  /* Total number of records */
+  ib_uint64_t total_recs = 0;
+
+  /* Total number of pages */
+  ib_uint64_t total_pages = 0;
+
+  /* Boundaries of the groups of distinct keys */
+  boundaries_t *n_diff_boundaries = nullptr;
+
+  /* Array of number of non-null keys for all prefixes */
+  ib_uint64_t *n_non_null_on_level = nullptr;
```

Comment:
Why is this pointer not declared next to `n_diff_on_level`? (And why they have a suffix `_on_level`?)